pkg/controller/operandrequest/config_status.go

```diff
@@ -27,8 +27,13 @@ import (
 func (r *ReconcileOperandRequest) updateServiceStatus(cr *operatorv1alpha1.OperandConfig, operatorName, serviceName string, serviceStatus operatorv1alpha1.ServicePhase) error {
 	klog.V(3).Info("Updating OperandConfig status")
 
-	cr.Status.ServiceStatus[operatorName].CrStatus[serviceName] = serviceStatus
-	if err := r.client.Status().Update(context.TODO(), cr); err != nil {
+	configInstance, err := r.getConfigInstance(cr.Name, cr.Namespace)
```

Comment:
@horis233 maybe we can add a wait logic until update success, you can followhttps://github.com/IBM/operand-deployment-lifecycle-manager/blob/e1467d496a45bf46d3551ecd676840160fd32fe9/test/helpers/helpers.go#L179

Reply:
@DanielXLee Thanks, I will add the wait logic. 
The reason for #212 is the resource version conflict. Current changes can fix it.
Adding wait logic can avoid the loss of status information.